pkg/test/integration/server.go

```diff
@@ -121,6 +140,7 @@ func defaultConfig() *Config {
 	}
 
 	return &Config{
+		RemoteCRDPaths:    []string{},
 		CRDDirectoryPaths: []string{},
```

Comment:
Do we need to make the distinction between remote and local CRD paths in our API? It seems like go-getter supports "getting" from the local filesystem.

Reply:
We could keep them all the same and then just copy anything local into the tmp dir as well, then pass that path to `envtest`. The reasoning for keeping them separate was that if no remote paths are supplied then we won't make a new tmp dir or do any operations, we will go straight to installing the local CRDs.

Reply:
My preference would be for what I feel is the cleaner UX of only taking one set of directory paths, even if it means we do some extra, unnecessary operations when only local CRDs are used.